types/pem/pem-tests.ts

```diff
@@ -558,13 +559,27 @@ const tests = {
 
           // test.ok(fs.readdirSync('./tmp').length === 0);
 
+          test.ok(Buffer.isBuffer(pkcs12.pkcs12));
+
           pem.readPkcs12(pkcs12.pkcs12, (error: any, keystore: any) => {
             test.ifError(error);
             test.ok(keystore);
 
             test.equal(ca.certificate, keystore.ca[0]);
             test.equal(cert.certificate, keystore.cert);
             test.equal(cert.clientKey, keystore.key);
+          });
+
+          const pkcs12File: string = __dirname + '/test.pkcs12';
+          fs.writeFileSync(pkcs12File, pkcs12.pkcs12);
+
+          pem.readPkcs12(pkcs12File, (error: any, keystore: any) => {
```

Comment:
Sorry should've seen this sooner. The reason the linter didn't balk the last few runs is because `keystore` is of type `any`, but it actually should match the newly added `Pkcs12ReadResult`. Can we change `any` here and the other usages (I see at least one at line 561)?

Reply:
thanks, changed this one as well